cardform/src/main/java/com/mercadolibre/android/cardform/presentation/viewmodel/InputFormViewModel.kt

```diff
@@ -225,7 +232,11 @@ internal class InputFormViewModel(
             tracker.trackEvent(
                 ErrorTrack(
                     TrackApiSteps.TOKEN.getType(),
-                    throwable.message.orEmpty()
+                    throwable.message.orEmpty(),
+                    binValidator?.bin!!,
+                    issuer?.id ?: 0,
+                    paymentMethod?.paymentMethodId!!,
+                    paymentMethod?.paymentTypeId!!
```

Comment:
```suggestion
                    binValidator?.bin.orEmpty(),
                    issuer?.id ?: 0,
                    paymentMethod?.paymentMethodId.orEmpty(),
                    paymentMethod?.paymentTypeId.orEmpty()
```
This could cause a NullPointerException